test/library/packages/UnitTest/ComplexInt.good

```diff
@@ -0,0 +1 @@
+./UnitTest.chpl:226: error: assert failed - 5 is not equal 7.0i
```

Comment:
Oh, I think this message should say "... is not equal *to* ..." or "5 *does* not equal..."